alyssa_mckee/string_list_assignments/foo_bar.py

```diff
@@ -0,0 +1,35 @@
+
+import time
```

Comment:
Very nice! I like seeing that you are checking to see how long it took for this function to run. This is a great tool to have moving forward into algorithms and checking to see which method is truly faster. +1